.github/workflows/experimental.yml

```diff
@@ -57,9 +57,15 @@ jobs:
           xmake build
 
       - name: Package
-        run: python tools/buildscripts/build.py package -e
+        id: package
+        run: python tools/buildscripts/release.py package -e
+
+      - name: Check for Package Step Failure
```

Comment:
The default behavior for github workflows is to give up and prevent further execution if any step fails. Am I missing the point of the manual checks?

Reply:
Just cos I thought that actions didn't automatically fail if the script does, but I was misinformed whenever that was (oops, I have been doing this for a while)